src/monitor/node_metadata.c

```diff
@@ -521,6 +588,51 @@ ReportAutoFailoverNodeHealth(char *nodeName, int nodePort,
 	SPI_finish();
 }
 
+/*
+ * ReportAutoFailoverNodeReplicationState persists the replication properties of
+ * a node.
+ *
+ * We use SPI to automatically handle triggers, function calls, etc.
+ */
+void
+ReportAutoFailoverNodeReplicationState(char *nodeName, int nodePort,
+							int candidatePriority,
+							bool replicationQuorum)
```

Comment:
We are not reporting the **state** of the node replication here, but the **settings** of the node replication, right? I think we should change the name of that function to reflect that.

Reply:
`ReportAutoFailoverNodeReplicationSetting`